Source/Diagnostics/ReducedDiags/BeamRelevant.cpp

```diff
@@ -161,7 +161,7 @@ void BeamRelevant::ComputeDiags (int step)
         if (w_sum < std::numeric_limits<Real>::min() )
         {
             for (int i = 0; i < static_cast<int>(m_data.size()); ++i){
-                m_data[i] = 0.0;
+                m_data[i] = amrex::Real(0.0);
```

Comment:
You should be able to do this, if you want to be more compact:
```suggestion
                m_data[i] = 0.0_rt;
```